go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/harvester/terraform-provider-harvester
 
-go 1.22
+go 1.22.5
 
 replace (
+	// From this commiet, it starts to support Darwin.
+	github.com/longhorn/go-common-libs => github.com/longhorn/go-common-libs v0.0.0-20240926084818-3a320d860af4
```

Comment:
Why do we need to specially have this replace directive? Don't we use the latest commit of go-common-libs as it's an indirect dependency and has the darwin arm64 support already?

@Yu-Jack

Reply:
@innobead  Actually, at that moment, we were using latest commit. This PR was open on Sep 27, 2024. The next commit was made on Oct 5, 2024. We had not made any commits when we merged this PR.

![image](https://github.com/user-attachments/assets/7530f5f7-8e44-4399-acf5-57dfd05f9fbd)

The reason we need this replacement is because of dependency chain. You can't directly remove the replacement, it results some dependency errors. In the meantime, you can't use latest commit either. Because latest go-common libs might be only designed to latest longhorn instead of old one.

Hence, if we'd like to bump this one, we need to spend time on checking dependency chain.